examples/dl-capsule/sqrl.py

```diff
@@ -1,6 +1,120 @@
 import torch
 import ksc.torch_frontend as knossos
 
+import ksc.compile
```

Comment:
Ideally these go below the "user-readable" knossos implementations, but I'm happy to do that as I document.